SPECS/wget/wget.spec

```
@@ -50,7 +65,7 @@ rm -rf %{buildroot}/%{_infodir}
 export PERL_MM_USE_DEFAULT=1
 cpan local::lib
 cpan HTTP::Daemon
-make  %{?_smp_mflags} check
+make %{?_smp_mflags} check || { cat "tests/test-suite.log"; exit 1 }
```

I think we should refrain from adding an explicit exit.
```suggestion
make %{?_smp_mflags} check || { cat "tests/test-suite.log"; false }
```

/cc @PawelWMS